content/docs/guides/self-hosted/api.md

```diff
@@ -61,3 +61,25 @@ The API service is a Go-based application. This is a single binary application t
 | SAML_CERTIFICATE_PRIVATE_KEY | Private key used by Pulumi to validate the SAML assertions sent by the IdP. Learn how to set this value [here]({{< relref "saml-sso" >}}). |
 | GITHUB_OAUTH_ENDPOINT | Used for GitHub API calls. |
 | GITLAB_OAUTH_ENDPOINT | Used for GitLab API calls. |
+
+### TLS Environment Variables
+
```

Comment:
I think it would be good to also call out if a self-signed certificate can be used for each of these touchpoints and a sample command showing how to generate one would be good too.

Reply:
ok, I added a note with an example of using openssl to generate self-signed certs in case they would like to use it.